cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -91,11 +91,17 @@ public class CruiseControl extends AbstractModel implements SupportsMetrics, Sup
     protected static final String TLS_CC_CERTS_VOLUME_MOUNT = "/etc/cruise-control/cc-certs/";
     protected static final String TLS_CA_CERTS_VOLUME_NAME = "cluster-ca-certs";
     protected static final String TLS_CA_CERTS_VOLUME_MOUNT = "/etc/cruise-control/cluster-ca-certs/";
+    protected static final String CRUISE_CONTROL_CONFIG_VOLUME_NAME = "config";
+    protected static final String CRUISE_CONTROL_SERVER_CONFIG_VOLUME_NAME = "server-config";
+    protected static final String CRUISE_CONTROL_CAPACITY_CONFIG_VOLUME_NAME = "capacity-config";
     protected static final String LOG_AND_METRICS_CONFIG_VOLUME_NAME = "cruise-control-metrics-and-logging";
-    protected static final String LOG_AND_METRICS_CONFIG_VOLUME_MOUNT = "/opt/cruise-control/custom-config/";
+    protected static final String CONFIG_VOLUME_MOUNT = "/opt/cruise-control/custom-config/";
     protected static final String API_AUTH_CONFIG_VOLUME_NAME = "api-auth-config";
     protected static final String API_AUTH_CONFIG_VOLUME_MOUNT = "/opt/cruise-control/api-auth-config/";
-
+    /**
+     * Name of the Cruise Control broker capacity config file
+     */
+    public static final String DEFAULT_CRUISE_CONTROL_CAPACITY_CONFIG_NAME = CONFIG_VOLUME_MOUNT + CRUISE_CONTROL_CAPACITY_CONFIG_VOLUME_NAME;
```

Comment:
This seems weird as the comment and what we do in other classes suggest this should be a filename and not path. So e.g. `capacity.proeprties` or `capacity.json` dpeending ont he format of the file.

Also, why is this public and defined here when it seems to be used only in different package? I guess that might apply to more of the constants here.

Reply:
Updated variable name to reflect what it is, and fixed the access